core/src/main/java/org/apache/iceberg/actions/RemoveFilesActionResult.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.actions;
+
+public class RemoveFilesActionResult implements RemoveFiles.Result {
+
+  private final long deletedDataFilesCount;
+  private final long deletedManifestsCount;
+  private final long deletedManifestListsCount;
+  private final long deletedOtherFilesCount;
+
+  public RemoveFilesActionResult(long deletedDataFilesCount,
+      long deletedManifestsCount,
```

Comment:
nit: formatting

We usually have like this:

<img width="596" alt="image" src="https://user-images.githubusercontent.com/6235869/117358836-7570ea00-ae6b-11eb-9db4-e48d8e801ab2.png">